src/Security/MemberAuthenticator/CMSLoginHandler.php

```diff
@@ -1,27 +1,29 @@
 <?php
 
-namespace SilverStripe\Security;
+namespace SilverStripe\Security\MemberAuthenticator;
 
 use SilverStripe\Control\HTTPResponse;
 use SilverStripe\Core\Convert;
+use SilverStripe\Security\CMSSecurity;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
 
-class CMSMemberLoginHandler extends MemberLoginHandler
+class CMSLoginHandler extends LoginHandler
 {
+    private static $allowed_actions = [
+        'LoginForm'
```

Comment:
Please ensure the case matches the method name. PSR-2 prefers lowerCamelCase, but I'll understand if for consistency you leave it as UpperCamelCase (since you'd need to make the change framework-wide).